src/Tribe/Attendees_Table.php

```diff
@@ -321,6 +321,7 @@ public function extra_tablenav( $which ) {
 			'TB_iframe' => true,
 			'width' => 410,
 			'height' => 300,
+			'parent' => 'admin.php',
```

Comment:
This change was to make sure the concerns in the other PR were addressed. I've noted that we should test https://central.tri.be/issues/46198 to make sure there are no regressions.